source/standards/logging.html.md.erb

```diff
@@ -1,6 +1,6 @@
 ---
 title: How to store and query logs
-last_reviewed_on: 2022-02-07
+last_reviewed_on: 2024-06-28
```

Comment:
This date could be updated to August, or even September - depending on when the PR is merged